packages/manager/apps/cloud/client/app/cloud/components/project/billing/archive-storage-list/billing-archive-storage-list.component.html

```diff
@@ -29,21 +29,21 @@
                 <td class="oui-datagrid__cell text-right" data-title="{{ :: 'cpbc_archive_storage_col_consumption' | translate }}">
                     <span>{{(storage.stored ? storage.stored.totalPrice : 0).toFixed(2) + ' ' + $ctrl.currencySymbol }}</span>
                     <i class="fa fa-question-circle left-space-m4"
-                       data-uib-tooltip="{{ :: $ctrl.getStorageVolumeInfoTooltip(storage) }}"
+                       data-oui-tooltip="{{ :: $ctrl.getStorageVolumeInfoTooltip(storage) }}"
                        tooltip-placement="left">
```

Comment:
```suggestion
                       oui-tooltip-placement="left">
```
I think you will have to also change the attribute responsible placement and other attributes that apply on tooltip configuration

Reply:
Since we're prefixing the custom attributes, it should even be `data-oui-tooltip-placement` :)